.github/workflows/publish.yml

```diff
@@ -0,0 +1,98 @@
+name: Publish nidaqmx
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: The environment to publish to.
+        default: 'none'
+        required: true
+        type: choice
+        options:
+          - none
+          - pypi
+          - testpypi
+
+env:
+  dist-artifact-name: nidaqmx-distribution-packages
+  environment: ${{ github.event_name == 'release' && 'pypi' || inputs.environment }}
+  environment-info: |
+    {
+      "pypi": {
+        "base-url": "https://pypi.org",
+        "upload-url": "https://upload.pypi.org/legacy/"
+      },
+      "testpypi": {
+        "base-url": "https://test.pypi.org",
+        "upload-url": "https://test.pypi.org/legacy/"
+      }
+    }
+
+jobs:
+  check_nidaqmx:
+    name: Check nidaqmx
+    uses: ./.github/workflows/build.yml
+  check_docs:
+    name: Check docs
+    uses: ./.github/workflows/generate_docs.yml
+  build_nidaqmx:
+    name: Build nidaqmx
+    runs-on: ubuntu-latest
+    needs: [check_nidaqmx, check_docs]
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Set up Python
+        uses: ni/python-actions/setup-python@e8f25e9a64426bd431ac124b83df11b76cdf60d5 # v0.1.0
+      - name: Set up Poetry
+        uses: ni/python-actions/setup-poetry@e8f25e9a64426bd431ac124b83df11b76cdf60d5 # v0.1.0
+      - name: Check project version
+        if: github.event_name == 'release'
+        uses: ni/python-actions/check-project-version@275868c1620d0343a006472f6300bd80f76bbd92 # users/bkeryan/update-project-version
```

Comment:
FYI this includes unreleased changes from https://github.com/ni/python-actions/pull/5

Reply:
Now it's released.